src/serac/physics/coefficients/coefficient_extensions.hpp

```diff
@@ -438,7 +438,6 @@ class AttributeModifierVectorCoefficient : public mfem::VectorCoefficient {
    * @param[in] v  The evaluated coefficient vector at the quadrature point
    * @param[in] Tr The element transformation for the evaluation
    * @param[in] ip The integration point for the evaluation
-   * @return The value of the coefficient at the quadrature point
```

Comment:
this change is unrelated to the title of the PR, but doxygen complained about it when I built the docs targets